src/components/AbsenceList/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from "@tanstack/react-query";
+import { fetchAbsences } from "../../services/api/apiCalls";
+import AbsenceItem from "../Absence";
+
+const AbsenceList = () => {
+  const { data } = useQuery({
+    queryKey: ["absences"],
+    queryFn: fetchAbsences,
+  });
+
+  return (
+    <div>
+      <ul className="list-none">
+        {data?.map((absence) => (
+          <li className="py-3" key={absence.id}>
```

Comment:
Again nice way of doing this, but when you map things you don't need to use a list all of the time. I would say a list is relevant when it is just one type of data, ie a shopping list. But in this case there is 5/6 types of data, so could be better structured in a table